src/Tizen.System.Information/Usage/ProcessMemoryUsage.cs

```diff
@@ -178,6 +182,98 @@ public int GetPrivateDirty(int pid)
+        /// <returns>The GPU memory size <paramref name="pid"/> is using (KiB).</returns>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="pid"/> is invalid.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int GetGPU(int pid)
+        {
+            if (Gpus == null)
+            {
+                Log.Error(InformationErrorFactory.LogTag, "No data");
+                InformationErrorFactory.ThrowException(InformationError.NoData);
+            }
+
+            for (int i = 0; i < Count; i++)
+                if (pid == Pids[i])
+                    return Gpus[i];
+
+            Log.Error(InformationErrorFactory.LogTag, "Invalid pid");
+            InformationErrorFactory.ThrowException(InformationError.InvalidParameter);
+            return 0;
+        }
+
+        /// <summary>
+        /// Gets the resident set size in graphic execution manager of a process.
+        /// </summary>
+        /// <param name="pid">The process id.</param>
+        /// <returns>The resident set size <paramref name="pid"/> is using (KiB).</returns>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="pid"/> is invalid.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int GetGemRss(int pid)
+        {
+            if (Gems == null)
+            {
+                Log.Error(InformationErrorFactory.LogTag, "No data");
+                InformationErrorFactory.ThrowException(InformationError.NoData);
+            }
+
+            for (int i = 0; i < Count; i++)
+                if (pid == Pids[i])
+                    return Gems[i];
+
+            Log.Error(InformationErrorFactory.LogTag, "Invalid pid");
+            InformationErrorFactory.ThrowException(InformationError.InvalidParameter);
+            return 0;
+        }
+
+        /// <summary>
+        /// Gets the SWAP memory size of a process.
+        /// </summary>
+        /// <param name="pid">The process id.</param>
+        /// <returns>The SWAP memory size <paramref name="pid"/> is using (KiB).</returns>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="pid"/> is invalid.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int GetSwap(int pid)
+        {
+            if (Swaps == null)
+            {
+                Log.Error(InformationErrorFactory.LogTag, "No data");
+                InformationErrorFactory.ThrowException(InformationError.NoData);
+            }
+
+            for (int i = 0; i < Count; i++)
+                if (pid == Pids[i])
+                    return Swaps[i];
+
+            Log.Error(InformationErrorFactory.LogTag, "Invalid pid");
+            InformationErrorFactory.ThrowException(InformationError.InvalidParameter);
+            return 0;
+        }
+
+        private void GetProcessMemoryValueInt(Interop.RuntimeInfo.ProcessMemoryInfoKey key, out int[] array)
```

Comment:
```suggestion
        private int[] GetProcessMemoryValueInt(Interop.RuntimeInfo.ProcessMemoryInfoKey key, IEnumerable<int> pids)
```
Do not use member variable `Pids` and `Count` in this method. This method is not thread safety. 
You'd better pass the `pids` as method argument.

And just return the `array`, don't use out parameter as much as possible.